FABulous/fabric_cad/bit_gen.py

```diff
@@ -116,8 +116,7 @@ def genBitstream(fasmFile: str, specFile: str, bitstreamFile: str):
         vhdl_str += '";\n'
     vhdl_str += "end package emulate_bitstream;"
 
-    # Legacy mode does not output configuration bits
-    # for the first and last row
+    # Legacy mode does not output configuration bit for the first and last row
```

Comment:
```suggestion
    # Legacy mode does not output configuration bits for the first and last row
```
Seems like the 's' got knocked off